docs/appendices/release-notes/5.10.6.rst

```diff
@@ -48,6 +41,16 @@ See the :ref:`version_5.10.0` release notes for a full list of changes in the
 Fixes
 =====
 
+- Fixed an issue that could lead to ``null`` values getting returned on
```

Comment:
For future reference, this fix was directly merged to 5.10 and not backported from master: https://github.com/crate/crate/pull/17844